docs/source/en/tasks/language_modeling.mdx

```diff
@@ -145,19 +145,26 @@ To apply the preprocessing function over the entire dataset, use 🤗 Datasets [
 ... )
 ```
 
-Now you'll need a second preprocessing function to capture text truncated from the lengthier examples to avoid losing any information. This preprocessing function should:
+This dataset contains the token sequences, but some of these are longer than the maximum input length for the model.
+You'll need a second preprocessing function to avoid losing any information.
+This preprocessing function should:
```

Comment:
This is not entirely correct (the sequence before wasn't either :sweat_smile: ). The goal is to convert a list of text in chunks of a certain block size (which might be equal to or lower than the maximum input length of the model). So the second preprocessing function is there to operate that conversion, not really to avoid losing information.

Reply:
Thanks for your comments @sgugger. I've removed the incorrect explanation. Let me know if there's anything else 😄